### PR TITLE
Wp 235 mock category data

### PR DESCRIPTION
**Pull request by jmhammons**
Adds a mock category sourced from https://www.meetup.com/meetup_api/docs/find/topic_categories/

Related:
- https://meetup.atlassian.net/browse/MW-654
- https://github.com/meetup/mup-web/pull/645/files#r100877257

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -173,3 +173,20 @@ export const MOCK_EVENT = {
 	visibility: 'public',
 	yes_rsvp_count: 23
 };
+
+// Mock category sourced from https://www.meetup.com/meetup_api/docs/find/topic_categories/
+export const MOCK_CATEGORY = {
```
Comment (jmhammons) [inline src/api.js:178]: So this is just a single category pulled from `/find/topic_categories/`. I waffled on making this an array, which is what the source endpoint returns. As is, tests will likely need to do `[MOCK_CATEGORY]` for this to be useful or similarly build an array to the desired length. Any thoughts, let me know.

Comment (sadafie) [reply src/api.js:178]: Maybe you can leave this as MOCK_CATEGORY, but also add a `export const MOCK_TOPIC_CATEGORIES = [MOCK_CATEGORY]`, that can be imported directly by the tests.

Comment (jmhammons) [reply src/api.js:178]: Yeah, good call @sadafie.

Comment (jmhammons) [reply src/api.js:178]: Clear to merge @sadafie?

Comment (sadafie) [reply src/api.js:178]: yep LGTM!

**Review by sadafie — APPROVED** [review]

**Pull request merged**